src/classes/Recipe.js

```diff
@@ -0,0 +1,35 @@
+class Recipe {
+    constructor(recipeDetails) {
+        this.id = recipeDetails.id;
```

Comment:
Would you add a default in case id doesn't get passed in?
`recipeDetails.id || [default id logic]`

Reply:
`
constructor(parameter){
this.id = parameter.id || generateRandomId();
...
}

generateRandomId(){
 //code goes here
 }`